src/Layers/xrRender/xrRender_console.cpp

```diff
@@ -852,6 +852,29 @@ class CCC_Fog_Reload : public IConsole_Command
 #   endif // MASTER_GOLD
 #endif // (RENDER == R_R3) || (RENDER == R_R4)
 
+class CCC_new_shader_support : public CCC_Mask
+{
+public:
+    CCC_new_shader_support(pcstr N, Flags32* V, u32 M) : CCC_Mask(N, V, M) { ; };
+    void Execute(LPCSTR args) override
+    {
+#if RENDER == R_R4
+        if (HW.FeatureLevel >= D3D_FEATURE_LEVEL_11_0)
```

Comment:
@Xottab-DUTY is this the correct way to check if we're using DX11 renderer? Seems to work okay on my end, but i wasn't sure.

Reply:
> @Xottab-DUTY is this the correct way to check if we're using DX11 renderer? Seems to work okay on my end, but i wasn't sure.

Yes, it is the correct way!